meta/runtime.yml

```diff
@@ -29,3 +31,11 @@ plugin_routing:
           deprecation:
             removal_version: 3.0.0
             warning_text: Use infoblox.bloxone.dns_view_info instead.
+        b1_dns_auth_zone:
+          deprecation:
+            removal_version: 3.0.0
+            warning_text: Use infoblox.bloxone.dns_auth_zone instead.
+        b1_dns_auth_zone_gather:
+          deprecation:
+            removal_version: 3.0.0
+            warning_text: Use infoblox.bloxone.dns_auth_zone_info instead.
```

Comment:
you also need to update the documentation of these resources

Reply:
Fixed